src/app/learn/learn.component.ts

```diff
@@ -6,7 +6,10 @@ import { Component } from '@angular/core';
 })
 export class LearnComponent {
   public sidebarRoutes: any[] = [
-    { name: 'Overview', path: '/learn' },
+    { name: 'Overview', path: '/learn', children: [
+      { name: 'Test1', path: '/learn/test-1' },
+      { name: 'Test2', path: '/learn/test-2' }
+    ]},
```

Comment:
I like this method. Need to make sure the `StacheNavLink` interface includes an option for `children?: StacheNavLink[]`.